README.md

```diff
@@ -11,7 +11,8 @@ Rather than being a standalone programming language, Halide is
 embedded in C++. This means you write C++ code that builds an
 in-memory representation of a Halide pipeline using Halide's C++
 API. You can then compile this representation to an object file, or
-JIT-compile it and run it in the same process.
+JIT-compile it and run it in the same process. Halide also provides
+Python bindings that mirror the C++ API.
```

Comment:
Perhaps "Halide also provides a Python binding that provides full support for writing Halide embedded in Python without C++."?

Reply:
Looks great. Thanks!